### PR TITLE
Fix #6464: PrimeIcons v7 constants

### LINKED ISSUE #6464: Missing icons
### Describe the bug

It seems the last batch of icons did not make it to PrimeReact... e.g. "gauge", "warehouse" and "hammer" are missing. In total, there are 311 icons, but only 261 types.

(Similar to: https://github.com/primefaces/primereact/issues/5333.)

### Reproducer

_No response_

### PrimeReact version

10.6.3

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

_No response_

### Steps to reproduce the behavior

_No response_

### Expected behavior

_No response_

### PR DESCRIPTION
**Pull request by melloware**
Fix #6464: PrimeIcons v7 constants

### CODE REVIEW

File: components/lib/api/PrimeIcons.js [hunk 1]
```diff
@@ -1,263 +1,316 @@
...
+    ANGLE_DOWN: 'pi .pi-angle-down',
+    ANGLE_LEFT: 'pi .pi-angle-left',
+    ANGLE_RIGHT: 'pi .pi-angle-right',
+    ANGLE_UP: 'pi .pi-angle-up',
+    APPLE: 'pi .pi-apple',
+    ARROW_CIRCLE_DOWN: 'pi .pi-arrow-circle-down',
+    ARROW_CIRCLE_LEFT: 'pi .pi-arrow-circle-left',
+    ARROW_CIRCLE_RIGHT: 'pi .pi-arrow-circle-right',
+    ARROW_CIRCLE_UP: 'pi .pi-arrow-circle-up',
+    ARROW_DOWN_LEFTANDARROW_UP_RIGHT_TO_CENTER: 'pi .pi-arrow-down-left-and-arrow-up-right-to-center',
```
Comment (inad9300) [inline components/lib/api/PrimeIcons.js:22]: I think there are some missing underscores around "AND" here.

Also, I count 311 icons in the documentation – how come 314 entries here?

Comment (inad9300) [reply components/lib/api/PrimeIcons.js:22]: @melloware It is "pi-circle-off", "pi-circle-on" and "pi-file-o" that are defined here but aren't listed in the documentation.

Comment (melloware) [reply components/lib/api/PrimeIcons.js:22]: I copied the names of the icons right from primeicons.css using regex and those values don't have underscores in the icon name so I kept them exact.

Comment (inad9300) [reply components/lib/api/PrimeIcons.js:22]: "ARROW_UP_RIGHT_AND_ARROW_DOWN_LEFT_FROM_CENTER" does have underscores, though, and reads much nicer.

File: components/lib/api/PrimeIcons.js [hunk 2]
```diff
@@ -1,263 +1,316 @@
...
-    VOLUME_OFF: 'pi pi-volume-off',
-    VOLUME_UP: 'pi pi-volume-up',
-    WALLET: 'pi pi-wallet',
-    WHATSAPP: 'pi pi-whatsapp',
-    WIFI: 'pi pi-wifi',
-    WINDOW_MAXIMIZE: 'pi pi-window-maximize',
-    WINDOW_MINIMIZE: 'pi pi-window-minimize',
-    WRENCH: 'pi pi-wrench',
-    YOUTUBE: 'pi pi-youtube'
+    ADDRESS_BOOK: 'pi .pi-address-book',
```
Comment (inad9300) [inline components/lib/api/PrimeIcons.js:2]: Uh, careful here! In the previous version, there was no period before the class name, but now there is.

Comment (melloware) [reply components/lib/api/PrimeIcons.js:2]: Ahh yeah I need to remove the period.

**Pull request merged**